.github/workflows/release.yml

```
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '**'
```

```suggestion
      - 'RELEASE_*'
```

Other tags would break the version-regex below